### PR TITLE
#25 - Active Instances endpoint

### LINKED ISSUE #25: Create ActiveInstances endpoint
We require an active instances read endpoint in order to pull out the active alerts from the API, this is required for website MVP.

### PR DESCRIPTION
**Pull request by Maelstromeous**
Closes #25

### CODE REVIEW

File: src/modules/data/entities/instance/instance.metagame.territory.entity.ts [hunk 1]
```diff
@@ -70,14 +71,14 @@ export default class InstanceMetagameTerritoryEntity {
...
-    @ApiProperty({example: 1, enum: ps2alertsEventStateArray, description: 'The internal event state. 0 = scheduled, 1 = in progress, 2 = finished'})
+    @ApiProperty({example: Ps2alertsEventState.ENDED, enum: ps2alertsEventStateArray, description: 'The internal event state. 0 = scheduled, 1 = in progress, 2 = finished'})
     @Column({
         type: 'enum',
         enum: ps2alertsEventStateArray,
     })
     state: Ps2alertsEventState;
 
-    @ApiProperty({description: 'Victory data for the instance'})
+    @ApiProperty({example: {vs: 34, nc: 33, tr: 33, cutoff: 0, winner: Faction.VANU_SOVEREIGNTY, draw: false}, description: 'Victory data for the instance'})
```
Comment (zhenghung) [inline src/modules/data/entities/instance/instance.metagame.territory.entity.ts:81]: I think it's better to put the example in the embed class instead.

File: src/modules/data/entities/aggregate/common/metagame.territory.result.embed.ts [hunk 2]
```diff
@@ -22,6 +23,11 @@ export default class MetagameTerritoryResultEmbed {
     })
     cutoff: number;
 
+    @Column({
+        type: 'number',
+    })
```
Comment (zhenghung) [inline src/modules/data/entities/aggregate/common/metagame.territory.result.embed.ts:28]: Not sure if this works but the other enums are written like
```
    @Column({
        type: 'enum',
        enum: factionArray,
    })
```

File: src/modules/data/entities/aggregate/common/metagame.territory.result.embed.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 /* eslint-disable @typescript-eslint/explicit-member-accessibility,@typescript-eslint/naming-convention */
 import {Column} from 'typeorm';
+import {Faction} from '../../../constants/faction.consts';
 
 export default class MetagameTerritoryResultEmbed {
```
Comment (zhenghung) [inline src/modules/data/entities/aggregate/common/metagame.territory.result.embed.ts:5]: Add swagger docs here (ApiProperty) for each column, It'll show up as the example so for the fields using the embed class.

**Review by zhenghung — COMMENTED** [review]
Mostly swagger stuff

File: src/modules/data/entities/instance/instance.metagame.territory.entity.ts (hunk 1 above)
Comment (Maelstromeous) [reply src/modules/data/entities/instance/instance.metagame.territory.entity.ts:81]: 3b410f2269ebc20d461ee4022d7e81f715fc642b

File: src/modules/data/entities/aggregate/common/metagame.territory.result.embed.ts (hunk 2 above)
Comment (Maelstromeous) [reply src/modules/data/entities/aggregate/common/metagame.territory.result.embed.ts:28]: 3b410f2269ebc20d461ee4022d7e81f715fc642b

File: src/modules/data/entities/aggregate/common/metagame.territory.result.embed.ts (hunk 3 above)
Comment (Maelstromeous) [reply src/modules/data/entities/aggregate/common/metagame.territory.result.embed.ts:5]: 3b410f2269ebc20d461ee4022d7e81f715fc642b

**Pull request merged**